scripts/gulp/tasks/gulpCypress.ts

```diff
@@ -172,7 +172,7 @@ export async function startCypressWatch () {
   }
 
   const watcher = chokidar.watch([
-    'packages/{graphql,data-context}/src/**/*.{js,ts}',
+    'packages/{graphql,data-context,scaffold-config}/src/**/*.{js,ts}',
```

Comment:
I was annoyed that making changes to `scaffold-config` never triggered the `Server update` toast so I'm including that watching here